requirements.txt

```diff
@@ -9,7 +9,8 @@ catalogue>=2.0.4,<2.1.0
 confection>=0.0.1,<1.0.0
 ml_datasets>=0.2.0,<0.3.0
 # Third-party dependencies
-pydantic>=1.7.4,!=1.8,!=1.8.1,<1.10.0
+pydantic>=1.7.4,!=1.8,!=1.8.1,<1.10.0; python_version < "3.7"
```

Comment:
I was confused by the specifically excluded versions here, but it looks like Pydantic has a `1.8` but no `1.8.0`. Just commenting for anyone else unfamiliar with this.